src/Cli/dotnet/commands/dotnet-workload/list/WorkloadListCommand.cs

```diff
@@ -26,6 +25,8 @@ internal class WorkloadListCommand : CommandBase
         private readonly IWorkloadInstallationRecordRepository _workloadRecordRepo;
         private readonly SdkFeatureBand _sdkFeatureBand;
 
+        public static readonly string MockUpdateDirectory = Path.Combine(Path.GetDirectoryName(Environment.ProcessPath),
```

Comment:
@wli3 are you already tracking replacing this mock update advertisement with a real implementation or should I add it to a issue?

Reply:
please add a new issue. I need to somehow make that work or just implement the real logic. Although that could be take some work considering we need to surface version and description information

Reply:
nvm, https://github.com/dotnet/sdk/issues/16484 has a separate list todo item